src/olympia/signing/tests/test_views.py

```diff
@@ -307,7 +307,7 @@ def test_system_addon_allowed(self):
         self.auto_sign_version.assert_called_with(latest_version)
 
     def test_system_addon_not_allowed_not_mozilla(self):
-        guid = 'systemaddon@mozilla.org'
+        guid = 'systemaddon@mozilla.com'
```

Comment:
hah, I think changing it from one domain in `SYSTEM_ADDON_GUIDS` to another is cheating :p
(I don't think its necessary to test each of them - the change just seemed unnecessary)

Reply:
We have multiple tests for different scenarios in there, so I figured might as well change the guid in one of them just to test with different things.